Documentation/helm-values.rst

```diff
@@ -1588,6 +1588,10 @@
      - Enable Google Kubernetes Engine integration
      - bool
      - ``false``
+   * - :spelling:ignore:`healthCheckICMPFailureThreshold`
+     - Number of ICMP requests sent for each health check before marking a node or endpoint unreachable.
+     - int
+     - ``1``
```

Comment:
Why is it `1` here and `3` elsewhere?
I'm comparing these files: 
- [Documentation/cmdref/cilium-agent.md](https://github.com/cilium/cilium/pull/36023/files#diff-26a87034ca4fd33b911176cd448b38fb1c636c2085fc87df2e78836c4b2600e9) – `default 3`
- [Documentation/helm-values.rst](https://github.com/cilium/cilium/pull/36023/files#diff-b9a59a825d08e17ba73f27f4d3eb3d0b5852c44ed80705c3003d51617237d1a6) – `1`
- [install/kubernetes/cilium/README.md](https://github.com/cilium/cilium/pull/36023/files#diff-82241bd6987061f9f739d835dd3377e0b820f7ee63cf265ef30177ff2a0cb403) – `1`
- [install/kubernetes/cilium/values.yaml](https://github.com/cilium/cilium/pull/36023/files#diff-0ea42ad21164b19bec1732225e254d3096d1e4040481c00053669287d81015fe) – `1`
- [install/kubernetes/cilium/values.yaml.tmpl](https://github.com/cilium/cilium/pull/36023/files#diff-aa0cf5fa6e9857c22d4fa74fb53d84cd302757b5ed8889c1111c32e0525230e2) – `3`
- [pkg/defaults/defaults.go](https://github.com/cilium/cilium/pull/36023/files#diff-873ba6ea91d97fbdbd851e2af37c2dc38cdd079dedef667a3a9dca4f457ae7e9) – `3`

Reply:
Good catch, I forgot to update the default helm value too. Gonna fix this :+1: